Makefile

```
@@ -265,6 +266,39 @@ cleanup-services: tools/generated/proxy.yml collection
+# --------------------------------------
+
+CI_IMAGE ?= quay.io/ansible/jewel-ci:$(shell git rev-parse --abbrev-ref HEAD)
+CI_CONTAINERFILE = tools/docker/Containerfile.ci
+CONTAINER_ENGINE ?= $(shell command -v podman 2>/dev/null || echo docker)
+
+## Build the CI container image (amd64 for GitHub Actions runners)
+ci-image:
+	$(CONTAINER_ENGINE) buildx build --platform linux/amd64 -f $(CI_CONTAINERFILE) -t $(CI_IMAGE) --load .
+
+## Build and push the CI container image (only from devel or stable-* branches)
+ci-image-push: ci-image
```

checkmake (0.3.2)

[warning] Target body for "ci-image-push" exceeds allowed length of 5 lines (19).

(maxbodylength)